indexing-service/src/main/java/io/druid/indexing/common/task/AbstractFixedIntervalTask.java

```diff
@@ -40,6 +40,16 @@ protected AbstractFixedIntervalTask(
 
   protected AbstractFixedIntervalTask(
       String id,
+      TaskResource taskResource,
```

Comment:
Good point, Nishant. I was hyper-focused on getting access to the "resource" node. :grinning:  It does make more sense to use the existing constructor. I can just modify IndexTask to do something similar to [RealTimeIndexTask] (https://github.com/OttoOps/druid/blob/index-add-taskResource/indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java#L116) for the groupId...